lesson 6

kuimovevgeniy opened the pull request:
Создан компонент search

kuimovevgeniy committed:
lesson 6

Создан компонент search

ubcent commented on public/app.js:
```diff
@@ -0,0 +1,62 @@
+Vue.component('search', {
+	template: `<input type="text" v-model="searchQuery">`
```
Он не знает что такое searchQuery, т.к. его нет в data этого компонента

ubcent replied:
Нужно его добавить в props, покажу на уроке как следовало сделать

kuimovevgeniy committed:
lesson 7

добавил метод удаления, но в консоле всеравно выдает ошибку и корзина не очищается полностью
